scripts/emoji_codegen.py

```diff
@@ -68,16 +105,67 @@ def generate_qml_list(**kwargs):
         code, qualification, charAndName = segments
 
         # skip unqualified versions of same unicode
-        if qualification == 'unqualified':
-            continue
-
-        if qualification == 'component':
+        if qualification != 'fully-qualified' and qualification != 'component' :
```

Comment:
Why did you change this condition?

Reply:
To allow skin tone modifiers to be displayed